bot-components/GitHub_types.mli

```diff
@@ -0,0 +1,95 @@
+type id = string
```

Comment:
The only problem with making `GitHub_types` an mli-only module is that we cannot abstract the `id` type (and if we don't, providing a type alias doesn't bring much). So in the end, we need to provide an ml implementation for this file or move the `id` type definition to another module.